picard/ui/util.py

```diff
@@ -81,15 +81,71 @@ def find_starting_directory():
     return find_existing_path(path)
 
 
-class MultiDirsSelectDialog(QtWidgets.QFileDialog):
+def picardize_caption(caption):
+    return _("Picard - %s") % caption
+
+
+def convert_filedialog_params(kwargs, default_caption):
+    # We use PySide/Qt syntax in code, so convert to PyQt6
+    if 'dir' in kwargs:
+        # TODO: remove with PySide
+        # pyside & qt use `dir`, pyqt uses `directory`
+        kwargs['directory'] = kwargs['dir']
+        del kwargs['dir']
+
+    if 'selectedFilter' in kwargs:
+        # TODO: remove with PySide
+        # pyside & qt use `selectedFilter`, pyqt uses `initialFilter`
+        kwargs['initialFilter'] = kwargs['selectedFilter']
+        del kwargs['selectedFilter']
+
+    # Ensure we have a caption in those dialogs
+    if 'caption' not in kwargs or not kwargs['caption']:
+        kwargs['caption'] = default_caption
```

Comment:
I think I never noticed this because Gtk does not show titles for file chooser dialogs. I checked and Windows uses the caption. Not sure about macOS, I think it might not have titles as well, but that needs to be checked.

Anyway, consistently setting a title makes sense.